stanfordnlp/dspy
# WIP - Testing on Notebooks - Refactor COT - Remove inheritance from predict and convert it to module.

What is the change?
- I have converted the COT to use the Predict instead of inheritance of predict and make it as module.

Why the change is important?
- We are in refactor phase that we wanna clean up stuffs and one of the part is to make sure anything is not inheriting from predict.

How to test?
- Run tests in the tests/ folder with pytest.
- Run notebooks to make sure they are fine.
